sources/api/shibaken/src/main.rs

```diff
@@ -45,19 +50,33 @@ enum Commands {
     /// Fetch and return whether or not this host is in the given partition.
     /// Accepts multiple partitions, returning `true` if the host is in any of the given partitions.
     IsPartition(partition::IsPartition),
+
+    /// Poll lifecycle state and wait until instance to be marked as InService
+    WarmPoolWait(warmpool::autoscaling_warm_pool::WarmPoolWait),
 }
 
 async fn run() -> Result<()> {
     let args: Args = argh::from_env();
 
-    // TerminalMode::Stderr will send all logs to stderr, as sundog only expects the json output of
-    // the setting on stdout.
-    TermLogger::init(
-        args.log_level,
-        LogConfig::default(),
-        TerminalMode::Stderr,
-        ColorChoice::Auto,
-    )
+    CombinedLogger::init(vec![
+        // Stderr terminal logger for settings generating subcommands, as sundog only expects the
+        // json output of the setting on stdout.
+        TermLogger::new(
+            args.log_level,
+            ConfigBuilder::new()
+                .add_filter_ignore_str("warmpool")
+                .build(),
+            TerminalMode::Stderr,
+            ColorChoice::Auto,
+        ),
+        // Simple logger for subcommands not called by sundog
+        SimpleLogger::new(
+            args.log_level,
+            ConfigBuilder::new()
+                .add_filter_allow_str("warmpool")
+                .build(),
```

Comment:
Shibaken was set up to send all logs to STDERR, including non-errors since it's called by sundog. Warm pool code isn't called by sundog, so we need log the normal way of STDOUT and STDERR.

Reply:
I wonder if just using `println!` in the warm pool code would be less brittle than relying on the logging filters? In the future, we'll need to remember to update these strings if we add or change subcommands.